packages/plugins/plugin-table/src/components/TableSection.tsx

```diff
@@ -10,7 +10,7 @@ import { ObjectTable, type ObjectTableProps } from './ObjectTable';
 
 const TableSection: FC<Omit<ObjectTableProps, 'getScrollElement'>> = ({ table }) => (
   <Table.Root>
-    <Table.Viewport classNames='max-bs-96 is-full sticky-top-0 !bg-[--surface-bg]'>
+    <Table.Viewport classNames='max-bs-96 is-full sticky-top-0 !bg-[--surface-bg] -ml-[1px] -mt-[1px]'>
```

Comment:
```suggestion
    <Table.Viewport classNames='max-bs-96 is-full sticky-top-0 !bg-[--surface-bg] -mis-px -mbs-px'>
```

Reply:
We need to standardize all of this of course, but good for now.

Reply:
What do you mean by standardize exactly? `px` is a “standard” length in Tailwind, and logical properties are meanginfully distinct from physical ones (see [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values)) and take precedence in the cascade.